scripts/py_matter_idl/matter_idl/generators/java/CHIPCallbackTypes.jinja

```diff
@@ -0,0 +1,40 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
```

Comment:
Nit, should we be doing 2023 now?

Reply:
Updated ... although I am not sure if it matters. Keeping track of years seems painful (I wish I could say "2nd millenium" and not worry about it anymore ... then again IANAL).